### ISSUE TITLE
Settings | General | Anntotation | Name form field lacks accessible name/label

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/readium-desktop#3492]
> screen reader reads “edit blank”

<img width="608" height="396" alt="Image" src="https://github.com/user-attachments/assets/87e50c0c-0829-4e37-a48a-8c4880894056" />

Submitted by University of Tennessee Libraries

**Comment by gautierchomel** [comment]
There is a label which is the name itself, once written, no label appears when no name exist. 
I think we need an empty state:
https://github.com/edrlab/thorium-reader/blob/f777851d30a8dc5b0858b627b4f6deb1816fc0f1/src/renderer/library/components/settings/Settings.tsx#L305-L309

Also I cannot find the `id` to which referes`name="creator-name"`, there is the `htmlFor="creator-name"` but I feel it's missing an `id` to make proper link.
